testsuite/model/src/test/java/org/keycloak/testsuite/model/KeycloakModelTest.java

```diff
@@ -290,6 +293,13 @@ public static KeycloakSessionFactory createKeycloakSessionFactory() {
         LOG.debugf("Creating factory %d in %s using the following configuration:\n    %s", factoryIndex, threadName, CONFIG);
 
         DefaultKeycloakSessionFactory res = new DefaultKeycloakSessionFactory() {
+
+            @Override
+            public void init() {
```

Comment:
Need to initialize Profile for model tests as it is no longer initialized by default, which is to prevent mistakes not using the correctly configured Profile for a given runtime.